src/main/java/cafeboard/Board/BoardService.java

```diff
@@ -0,0 +1,42 @@
+package cafeboard.Board;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@Service
+public class BoardService {
+    private final BoardRepository REPOSITORY;
```

Comment:
여기도 일반적으로 이름을 아래처럼 짓습니다.
```suggestion
    private final BoardRepository boardRepository;
```